lib/unit-ruby/util/connection.rb

```diff
@@ -59,6 +63,19 @@ def patch(path, data = nil)
       from_json_api(response.body)
     end
 
+    # Executes a PUT of a JSON object without transforming the keys
+    def put_json(path, data = nil)
+      response = connection.put do |req|
+        req.url path
+        req.headers['Content-Type'] = 'application/vnd.api+json'
+        req.body = data.deep_transform_keys!(&:to_s) if data
```

Comment:
I would just assume we’d already pass in valid json to a method called put_json?

I’m trying to figure out if we can keep this consistent with the existing methods (either by reusing the post, but passing in something like transform: false; or adding our own put method). Is it a smell that we have the resources representation (json vs hash) this deep in Connection?